linux/scripts/deb-packaging.sh

```diff
@@ -11,14 +13,18 @@ THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
 
 builder_describe \
   "Helper for building Debian packages." \
-  "dependencies              Install dependencies as found in debian/control" \
-  "source+                   Build source package" \
-  "verify                    Verify API" \
-  "--gha                     Build from GitHub Action"
+  "dependencies               Install dependencies as found in debian/control" \
+  "source+                    Build source package" \
+  "verify                     Verify API" \
+  "--gha                      Build from GitHub Action" \
+  "--bin-pkg=BIN_PKG          Path and name of binary Debian package (for verify action)" \
+  "--pkg-version=PKG_VERSION  The version of the Debian package (for verify action)" \
+  "--git-ref=GIT_REF          The ref of the HEAD commit, e.g. HEAD of the PR branch (for verify action)" \
+  "--git-base=GIT_BASE        The ref of the base commit, e.g. HEAD of the master branch (for verify action)"
```

Comment:
Should these be SHAs or commitishes/refs, tag names, branch names or what? :grin:

Reply:
```suggestion
  "--git-ref=GIT_REF          The SHA of the HEAD commit, e.g. of the PR branch (for verify action)" \
  "--git-base=GIT_BASE        The ref of the base commit, usually the name of the base branch, e.g. master, stable-16.0 (for verify action)"
```